tests/console/validate_lvm_with_raid1.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's openQA tests
```

Comment:
the name of the file as mentioned: `validate_lvm+raid1.pm`

Reply:
done